common/types/shared.go

```diff
@@ -5572,6 +5688,50 @@ type SignalWithStartWorkflowExecutionRequest struct {
 	JitterStartSeconds                  *int32                 `json:"jitterStartSeconds,omitempty"`
 }
 
+func (v *SignalWithStartWorkflowExecutionRequest) BodyToString() string {
+	cloneRequest := SignalWithStartWorkflowExecutionRequest{
```

Comment:
Add a comment here why you are manually assigning attributes. You are trying to prevent logging workflow params, i.e. user inputs.

Reply:
done